canonical/ubuntu.com
# /download/intel-iei-tank-870 page refresh

## Done

- Applied refresh as per [design](https://www.figma.com/file/cVGimTPHky4h3BPXfrT6GB/24.04-U.com---download-(rebranding)?node-id=246%3A3249&mode=dev), [doc](https://docs.google.com/document/d/1Fa2JPxlEmZTyaFL_OSaQ-_OFW67AtBB2F-652TETqRk/edit), and [live page](https://ubuntu.com/download/intel-iei-tank-870)
  - Content should match the live page except where clear changes have been made on the copy doc (suggestion mode) 

## QA

- View the site locally in your web browser at: https://ubuntu-com-13576.demos.haus/download/intel-iei-tank-870
- Check against design, live page, and doc

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-8797
